container/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM nixpkgs/nix:latest@sha256:79397901292ed23c3502177552e4d261c2d711720185e2b3a5beb715f35f595b
+
+ARG CACHIX_VERSION=v1
+ARG DEVENV_VERSION=latest
+
+# Add Nix to PATH
+ENV PATH="/nix/var/nix/profiles/default/bin:$PATH"
+
+# https://github.com/NixOS/nix/issues/5258
+RUN mkdir -p -m 0755 /etc/nix
+RUN echo "filter-syscalls = false" > /etc/nix/nix.conf
+
+RUN nix-env -iA cachix -f https://cachix.org/api/$CACHIX_VERSION/install && cachix use devenv
```

Comment:
@sandydoo @domenkozar I wonder if it actually makes sense to install cachix here as the purpose I see on this container is enabling devenv on container-based stack like Gitlab CI, AWS Batch, Kubernetes, etc.

From those places this container will simply provision environments to run automated scripts, meaning that no actual developers will be using it.

Reply:
Given that cachix ships with devenv, I don't think it's needed

Reply:
It looks like building the container without cachix takes a lot longer as compilation is needed. I will leave cachix available so cache for both devenv can be fetched

Reply:
Also hit a segfault when building without cache: https://github.com/dsalaza4/devenv/actions/runs/13637371548/job/38119335111#step:7:5507